app/src/androidTest/java/org/simple/clinic/overdue/AppointmentRepositoryAndroidTest.kt

```diff
@@ -1818,6 +1821,228 @@ class AppointmentRepositoryAndroidTest {
     assertThat(overdueAppointments).containsExactlyElementsIn(expectedAppointments)
   }
 
+  @Test
+  fun patients_with_different_combinations_of_blood_pressure_and_blood_sugars_should_be_included_in_overdue_list() {
```

Comment:
We can maybe add blood sugar to the existing test `high_risk_patients_should_be_present_at_the_top_when_loading_overdue_appointments` instead of a new test, wdyt?

Reply:
I tried doing this, but the function did not have a way to add blood sugars. Also, the test is already quite long I feel.